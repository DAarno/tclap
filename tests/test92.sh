#!/bin/bash

../examples/test29 -a 5 -b 7 > tmp.out 2>&1

if cmp -s tmp.out $srcdir/test92.out; then
	exit 0
else
	exit 1
fi
