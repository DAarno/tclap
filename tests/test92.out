PARSE ERROR: Argument: -a AND -b provided.
             Only one is allowed.

Brief USAGE: 
   ../examples/test29  [-a int|-b int|-s str] [--version] [-h]

For complete USAGE and HELP type: 
   ../examples/test29 --help

